src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1255,6 +1286,14 @@ impl_types layout_optimizer::get_preferred_impl_type(program_node& node, format
         if (input_layout.format.dimension() != output_layout.format.dimension()) {
             preferred_impl = impl_types::ocl;
         }
+
+        // For mixed precision case, onednn is slower than cldnn
+        if (input_layout.format == format::b_fs_yx_fsv16 && (input_layout.data_type == data_types::u8 || input_layout.data_type == data_types::i8))
```

Comment:
Please use local variables with short names like `input_fmt` (defined in a code line 1265), `input_dt` in this new block of the code and in the code above which is related to `node.is_type<reorder>` and which you didn't change.

Reply:
Applied, thanks. I defined `data_type_traits::is_i8_u8` as a helper function, too.